cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
this comes from the cypress initialiser? doesn't seem to be used anywhere.

Reply:
yep, cypress upgrade added it. probably we should remove it just wanted to avoid the need to remove after every cypress uograde. we should look into it sometime

Reply:
could you remove? or it breaks something?